include/dp_flow.h

```diff
@@ -87,11 +87,9 @@ struct flow_nf_info {
 	uint32_t vni;
 	uint16_t icmp_err_ip_cksum;
 	enum dp_flow_nat_type nat_type;
-	uint8_t underlay_dst[16];
+	union dp_ipv6 underlay_dst;
 	uint8_t l4_type;
-} __rte_packed;
-static_assert(sizeof(((struct flow_nf_info *)0)->nat_type) == 1,
-			  "enum dp_flow_nat_type is unnecessarily big");
```

Comment:
Packing removed because the only use of this structure is in `struct flow_value` which is not packed.